backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/age/Age.java

```diff
@@ -0,0 +1,77 @@
+package de.metas.handlingunits.age;
+
+/*
+ * #%L
+ * metasfresh-pharma
+ * %%
+ * Copyright (C) 2018 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+import lombok.EqualsAndHashCode;
+import lombok.NonNull;
+import lombok.Value;
+
+@Value
+@EqualsAndHashCode
+public final class Age
+{
+	private final Integer ageInMonths;
+
+	public static Age ZERO = ofAgeInMonths(0);
+
+	@NonNull
+	public static Age ofAgeInMonths(final Integer ageInMonths)
+	{
+		return new Age(ageInMonths);
```

Comment:
why Integer and not `int`?

also here you could check: if (ZERO.ageInMonths == ageInMonths) return ZERO;